### PR TITLE
Issues/269 sqla reimport

### PR DESCRIPTION
**Pull request by jessesnyder**
Fix to prevent SQLAlchemy warnings (see [issue 269](https://github.com/Dallinger/Dallinger/issues/269))

## Description
Move all SQLAlchemy ORM class definitions out of the `experiment` modules, so the `experiment` module can be loaded multiple times via `imp.load_module()` without triggering warnings from SQLAlchemy's `Mapper` class.

## Motivation and Context
Bug fix

## How Has This Been Tested?
All demos have been run with the changes in place.

### CODE REVIEW

File: demos/mcmcp/experiment.py [hunk 1]
```diff
@@ -170,6 +68,7 @@ class Perturbation(Transformation):
 
 @extra_routes.route("/choice/<int:node_id>/<int:choice>", methods=["POST"])
 def choice(node_id, choice):
+    from sources import Agent
```
Comment (suchow) [inline demos/mcmcp/experiment.py]: The MCMCP demo has several kinds of custom objects: `Info`, `Transformation`,     `Source`, and `Agent`, so I don't think putting these all in a `sources.py` file is appropriate. I think there are two possible fixes: one is to separate these into four separate files; the other is to rename `sources.py` to something more generic like `extra_models`.

File: demos/rogers/experiment.py [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
+        The sources module is imported here because it must be imported at
+        runtime.
+
+        A few properties are then overwritten.
+
+        Finally, setup() is called.
+        """
         super(RogersExperiment, self).__init__(session)
+        import sources
+        self.sources = sources
```
Comment (suchow) [inline demos/rogers/experiment.py]: (The same comment I left on the MCMCP demo applies here. With so many custom objects, renaming to something more generic is probably the way to go.)

Comment (jessesnyder) [inline demos/rogers/experiment.py]: How about just `models`?

Comment (suchow) [inline demos/rogers/experiment.py]: @jessesnyder `models` is good!

**Pull request merged**